infra/general/kHtmlPurifier.class.php

```diff
@@ -84,6 +84,9 @@ public static function initHTMLPurifier()
 		{
 			$config = HTMLPurifier_Config::createDefault();
 			$config->set('Cache.DefinitionImpl', null);
+			$defaultAllowedHtmlTags = 'p[],img[src,title,alt],a[href,rel,target],span[class],div[],br[],b[],i[],u[],ol[],ul[],li[],blockquote[]';
+			$allowedHtmlTags = kConf::get('html_purifier_allowed_html_tags', $defaultValue=$defaultAllowedHtmlTags);
```

Comment:
Why do you need this assignment $defaultValue=$defaultAllowedHtmlTags)
Just send $defaultAllowedHtmlTags

Reply:
true.. will fix it